test/workspace/test.vue

```diff
@@ -3,17 +3,12 @@ type ServerReadinessProbe = {
   a?: string;
 };
 
-
 type StringPrimitive = string;
 type TestPrimitiveObj = { value: StringPrimitive };
 </script>
 
 <template>
-  <div>
-
-  </div>
+  <div></div>
```

Comment:
There sadly some changes like this here and there. The reason is that I accidentally ran prettier for more files that I though - when trying to repair linting issues.